### PR TITLE
feature/payment

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# Features added : 

- [x] Add dummy Razorpay gateway.

Click on place order to view Razorpay gateway.
*Note you have to login using test credentials*
[Preview Link](https://football-ecommerce-git-feature-profile-payment-kevin-solomon.vercel.app/)

### CODE REVIEW

File: src/pages/Signup/Singup.jsx [hunk 1]
```diff
@@ -30,7 +30,11 @@ function Singup() {
         email,
```
Comment (mohammed-sas) [inline src/pages/Signup/Singup.jsx:30]: typo "Signup.jsx"

**Review by mohammed-sas — CHANGES_REQUESTED** [review]
on successful payment the cart should reset

**Review by mohammed-sas — APPROVED** [review]
Payment functionality is working as expected
Code is clean and modular
PR is good to merge

**Comment by starver20** [comment]
Adding item to cart from single-product view is giving some error.
You can redirect to homepage after order is successfully placed.
Cart and wishlist icon on the navbar are not visible.

**Review by starver20 — APPROVED** [review]
Other than that, the pr looks good to merge. Good work.

**Comment by Kevin-Solomon** [comment]
> 

Yup using Font Awesome is a bit unreliable as the network request gives an error sometimes.
Will switch to react icons before submission.

**Pull request merged**